Source/WebCore/platform/graphics/FontSelectionAlgorithm.h

```diff
@@ -433,11 +426,6 @@ struct FontSelectionCapabilities {
     Range slope { normalItalicValue() };
 };
 
-constexpr bool operator==(const FontSelectionCapabilities& a, const FontSelectionCapabilities& b)
-{
-    return a.tied() == b.tied();
```

Comment:
Maybe we should delete the `tied` function too and just move the code into the `add(Hasher&)` function?